### DISCUSSION TITLE
discussion에도 사진 넣을 수 있나요??

### DISCUSSION THREAD

**Discussion by kmouleejunhyuk** [boostcampaitech2/object-detection-level2-cv-06#13] [Q&A] [answered]
노션에는 사진을 넣었는데 디스커션도 가능하려나요

**Comment by ppskj178** [comment, marked as answer]
가능해요 슬랙에 제가 올린 글 참고해주세요
![image](https://user-images.githubusercontent.com/20790778/135573220-84f2b92c-fa95-4225-9986-da5ae0f8feb4.png)
